src/fixtureParser.ts

```diff
@@ -26,17 +30,36 @@ const removeTrailingPytestInfo = (lines: string[]) => {
     return lines.slice(0, lastFixture);
 };
 
+/**
+ * Find where pytest is installed and return the pip package location
+ */
+
+const findPipPackageLocation = async () => {
+    let pyTestLocation = "";
+    const response = spawnSync(PythonShell.defaultPythonPath, ["-m", "pip", "show", "pytest"]);
```

Comment:
Can you use [this](https://github.com/nickjmiller/pytest-fixtures/blob/main/src/fixture.ts#L9) instead to get the python path? I am worried this will ignore user workspace settings.